client/src/components/ItemForm.js

```diff
@@ -40,27 +41,46 @@ const ItemForm = () => {
           if (menuItem.message) {
             alert(`${menuItem.message}`);
           } else {
-            result.setMenuItems([...result.menuItems, menuItem]);
-            name.current.value = '';
-            price.current.value = '';
+            result.setData([...result.data, menuItem]);
+            form.name = '';
+            form.price = '';
```

Comment:
why are we doing this when form is defined with useState?

Reply:
You might be viewing an older commit? That line doesn't exist anymore.